### ISSUE TITLE
GCP Provider instance is "invalid"

### ISSUE THREAD

**Issue by jbw976** [crossplaneio/crossplane#102]
GCP provider in our example yaml is getting set by its controller as "invalid" due to no scopes being specified when performing the validity testing.  Note that the provider object can still be used to use the GCP API, for instance to create CloudSQL instances, but it is being marked as invalid.

```
Post https://cloudresourcemanager.googleapis.com/v1/projects/long-disk-191221:testIamPermissions?alt=json&prettyPrint=false: oauth2: cannot fetch token: 400 Bad Request, Response: {"error": "invalid_scope","error_description": "Empty or missing scope not allowed."}
```

Example YAML:
https://github.com/upbound/conductor/blob/master/cluster/examples/wordpress/gcp/class/provider.yaml#L12-L24

**Comment by negz** [comment]
I believe this will be fixed by https://github.com/crossplaneio/crossplane/pull/500.

**Comment by negz** [comment]
This was 'fixed' by #500 in that there is no longer a GCP provider controller. :)
